Sources/BraintreeCoreSwift/BTPreferredPaymentMethods.swift

```diff
@@ -0,0 +1,76 @@
+import UIKit
+
+///  :nodoc:
+///  Fetches information about which payment methods are preferred on the device.
+///  Used to determine which payment methods are given preference in your UI, not whether they are presented entirely.
+///  This class is currently in beta and may change in future releases.
+@objcMembers public class BTPreferredPaymentMethods: NSObject {
+
+    // MARK: - Internal Properties
+
+    var application: AnyObject
+
+    private let apiClient: BTAPIClient
+
+    // MARK: - Initializer
+
+    /// Creates an instance of BTPreferredPaymentMethods.
+    /// - Parameter apiClient: An API client
+    @objc(initWithAPIClient:)
+    public init(apiClient: BTAPIClient) {
```

Comment:
We should probably add `nodoc` to all of the funcs on this class to be safe

Reply:
Updated here: https://github.com/braintree/braintree_ios/pull/890/commits/57cdffb3685c054df240490b628e53b925549b72